apps/backend/src/routers/news.ts

```diff
@@ -1,5 +1,5 @@
 import { router, procedure } from '../trpc';
-import NewsModel, { News } from '$models/News';
+import NewsModel, { News } from '../models/News';
```

Comment:
Long story short, aliases will cause TS to break when importing the server's type definition into the frontend, since the frontend's tsconfig doesn't have these aliases.